go/api/config.go

```diff
@@ -0,0 +1,323 @@
+/**
```

Comment:
Silly question, seems like license headers in Go usually use // instead of /* */, like [this](https://github.com/go-sql-driver/mysql/blob/master/atomic_bool.go#L1). Should I change to use //?